gpbft/participant.go

```diff
@@ -28,6 +29,16 @@ type Participant struct {
 	terminatedDuringRound uint64
 }
 
+type validatedMessage struct {
+	msg *GMessage
+}
+
```

Comment:
I ended up duplicating this structure two more times in tests. I could instead just make it a public exported type in the GPBFT package. This would enable trivial bypassing of message validation. I'm unsure of its potential for accidental use.

Reply:
Alternatively you can move the tests that use it to `gpbft` package instead of `gpbft_test`.